provider/elastictranscoder/aws.go

```diff
@@ -289,7 +289,7 @@ func (p *awsProvider) JobStatus(job *db.Job) (*provider.JobStatus, error) {
 	if len(resp.Job.Outputs) > 0 {
 		statusMessage = aws.StringValue(resp.Job.Outputs[0].StatusDetail)
 		if strings.Contains(statusMessage, ":") {
-			errorMessage := strings.Split(statusMessage, ":")[1]
+			errorMessage := strings.SplitN(statusMessage, ":", 2)[1]
 			statusMessage = strings.Trim(errorMessage, " ")
```

Comment:
Oh, just saw this too. You can use ``strings.TrimSpace`` to safely support any kind of spaces :D